src/app/auth/components/auth/auth.component.html

```diff
@@ -1,12 +1,28 @@
-<div class="perm-banner">
-  <div class="banner-content">
-  <a href="/">
-    <img class="banner-logo" src="assets/img/logo/sideways_logo.png">
-  </a>
+<!-- @format -->
+<div class="auth-container">
+  <div class="image">
+    <div class="testimony">
+      <p class="text">
+        At Permanent, we celebrate our members’ hard work through our public
+        gallery and archive spotlights. Explore our public archive spotlights
+        and, when you’re ready, publish or share your own public archive!
+      </p>
+      <a
+        class="link"
+        href="https://permanent.org/blog/tag/public-archive-spotlight/"
+        >Start Exploring Now.</a
+      >
+    </div>
+    <img src="assets/img/sign_in_image.png" />
```

Comment:
This should probably have an `alt` attribute. Either with a blank alt if we decide this is a purely decorative image, or proper alt text if we want to preserve that.

Reply:
@k8lyn6 is there a specific alt text you might consider using?